custom/icds_reports/templates/icds_reports/icds_app/download.directive.html

```diff
@@ -119,7 +119,7 @@
                            popover-placement="top-right"
                            popover-trigger="'mouseenter'"></i></label>
                     <div class="col-sm-6">
-                        <ui-select ng-model="$ctrl.selectedDataPeriod" theme="bootstrap">
+                        <ui-select ng-model="$ctrl.selectedDataPeriod" theme="bootstrap" on-select="$ctrl.onDataPeriodSelect($item)">
```

Comment:
why do you need both here? The Model as well and on-select callback as well?

Reply:
to update the year function

Reply:
What is being updated in onDataPeriodSelect yeah? why is model still needed?

Reply:
offlined, I think we need this.